docgen/README.md

```diff
@@ -25,6 +25,29 @@ command=docgen
 out=examples/util/v1/
 ```
 
+### Code examples
+
+To generate code examples, add the following to the `.gunkconfig` docgen section:
+
+```ini
+lang=go
+```
+
+Then add your `*.go` files near your gunk files.
+The examples files must be named according to the gunk method you want to showcase.
+
+Example:
+
+```go
+// UpdateAccount updates an account.
+UpdateAccount(UpdateAccountRequest)
+
+// DeleteAccount deletes an account.
+DeleteAccount(DeleteAccountRequest)
+```
+
+You should have `UpdateAccount.go` and `DeleteAccount.go`.
```

Comment:
Since these are not real go files I didn't think much about them and followed the pulpMd example. 

I've converted the function name using snaker in the template func, files should now be name `get_account.go`